src/core/scene/a-scene.js

```diff
@@ -320,7 +364,16 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+        // Postproc must be explicitly enabled by a component or system.
```

Comment:
you mean having something like the debug component and checking for it's presence? In the current design there is a post-process component who's purpose is to pass the order of application for the post processing effects to the compositor system as depicted here http://wizgrav.github.io/aframe/examples/test/post-processing-simple/ . Mind that the aframe.js used by the example is patched with the updated VREffect that works with actual hmds (not with the higher fps though)